pkg/kubelet/apis/podresources/server_v1_test.go

```diff
@@ -754,15 +756,15 @@ func equalListResponse(respA, respB *podresourcesapi.ListPodResourcesResponse) b
 				return false
 			}
 
-			if !euqalDynamicResources(cntA.DynamicResources, cntB.DynamicResources) {
+			if !equalDynamicResources(cntA.DynamicResources, cntB.DynamicResources) {
 				return false
 			}
 		}
 	}
 	return true
 }
 
-func euqalDynamicResources(draResA, draResB []*podresourcesapi.DynamicResource) bool {
+func equalDynamicResources(draResA, draResB []*podresourcesapi.DynamicResource) bool {
```

Comment:
since we are cleaning up things, please separate this function from `equalContainerDevices` adding a newline between lines 805 and 806

Reply:
done